src/cascadia/TerminalApp/MinMaxCloseControl.xaml

```diff
@@ -62,13 +62,26 @@ the MIT License. See LICENSE in the project root for license information. -->
             <x:String x:Key="CaptionButtonPath"></x:String>
             <x:String x:Key="CaptionButtonPathWindowMaximized"></x:String>
 
+            <!-- "CaptionButtonHeightWindowed" and
+                 "CaptionButtonHeightMaximized" define the size we should use
+                 for the caption buttons height for the windowed and maximized
+                 states, respectively.
+
+                 32 was chosen for the Maximized height to match the height of
+                 the TabRowControl. This way, when the window is maximized, the
+                 tabs will be flush with the top of the window. See GH#2541 for
+                 details.-->
+            <x:Double x:Key="CaptionButtonHeightWindowed">36.0</x:Double>
+            <x:Double x:Key="CaptionButtonHeightMaximized">32.0</x:Double>

```

Comment:
we can probably bind this to a tabview height resource! check TabView.theme.xaml. If one doesn't exist, then yeah we should just hardcode.

check on different dpis! since these are in dip.

Reply:
There's not really a defined height, it's just an aggregate of heights:
https://github.com/microsoft/microsoft-ui-xaml/blob/98a6b746dc409fa3c64bca4db2c3e6aa31e73470/dev/TabView/TabView_themeresources.xaml

Reply:
Also definitely just checked different DPIs and they were fine ☺️